javascript/src/domain/agents/__tests__/agent-shapes-export-surface.test.ts

```
@@ -0,0 +1,83 @@
+ *
+ * Nothing asserted that before. The shim's presence was the only thing making
+ * the old import path work, and its absence would have been a silent breaking
+ * change for any consumer importing from the `voice` namespace. So this file
+ * asserts the surface rather than the implementation: it does not care where
+ * the declarations live, only that both entry points still carry them.
+ */
+
+import { describe, expect, it } from "vitest";
+
+import * as domainAgents from "../index";
+import * as voice from "../../../voice/index";
```

ESLint

[error] `../../../voice/index` import should occur before import of `../index`

(import/order)